.github/workflows/wpt.yml

```diff
@@ -235,9 +234,6 @@ jobs:
     name: Commit updated expectations
     runs-on: ubuntu-latest
     needs: [update_expectations]
-    # Experiment.
-    # TODO: remove.
-    if: success() || failure()
```

Comment:
Are you sure this job will work if the main WPT job failed?

Reply:
I think we can check the `update_expectations` that it succeed.

Reply:
Apparently it does not: https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/13078541005/job/36496854875

Reply:
I will take a look.

Reply:
Fixed in https://github.com/GoogleChromeLabs/chromium-bidi/pull/3062